frontend/src/pages/Contracts/Contract/Deliverables/tableConfig.tsx

```diff
@@ -7,7 +8,7 @@ export const tableConfig = () => {
     { field: "fiscal_year", headerName: "Fiscal", flex: defaultFlex },
     { field: "description", headerName: "Description", flex: defaultFlex },
     { field: "is_expense", headerName: "Is Expense", flex: defaultFlex },
-    { field: "completion_date", headerName: "Completion Date", flex: defaultFlex },
+    { field: "completion_date", headerName: "Completion Date", valueFormatter, flex: defaultFlex },
```

Comment:
It's less code this way but I feel like it kind of obscures what's going on here. I think the full `valueFormatter: dateFormatter` would be easier for me to understand at a glance.

Reply:
yeah i could see that